apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -183,6 +183,7 @@ export function HeaderNav({}: Props) {
           <NotificationCenterWidget user={currentUser} />
           <Dropdown
             position="bottom-end"
+            styles={{ dropdown: { minWidth: 220 } }}
```

Comment:
moved this style from the `Dropdown` component to this specific for that style place - user header dropdown